mk/verilator.mk

```diff
@@ -1,12 +1,15 @@
-
 include mk/common.mk
 
 default: lint
 
 .PHONY: lint
-lint: $(addprefix ${OUTDIR}/,$(addsuffix .verilator_lint.txt,${TESTCASE_NAMES}))
+lint: $(addprefix ${OUTDIR}/,$(addsuffix .verilator_lint.stdout,${TESTCASE_NAMES}))
+
+# See Verilator maunual, Release Devel 5.009, Wilson Snyder 2023-03-18
+# --lint-only: Lint the design - check for warnings & errors; no output file.
+#  Pages 16(19) & 77(80).
+# -Wall: Enable all warning and writing style messages (enables stronger linting).
+#  Pages 85(88), 2(5) & 68(71).
```

Comment:
The pdf viewer had different page numbers. I thought that might be helpful. I can keep the page numbers of the manual instead to avoid the confusion. No the & here is 'and'.

Reply:
Would you either add a comment to the PR saying that, or just use one set of page numbers and list style, please.

Reply:
Changes effected, please. Ok to merge.